test/image/mocks/gl3d_line-colorscale-with-markers.json

```diff
@@ -0,0 +1,15 @@
+{
+  "data": [
+    {
+      "type": "scatter3d",
+      "x": [0, 0, 0],
+      "y": [1, 1, 1],
+      "z": [1, 2, 3],
+      "mode": "lines+markers",
+      "line": {
+        "color": [0, 1, 2],
+        "width": 10
+      }
+    }
+  ]
+}
```

Comment:
@etpinard Would you please test the image and hover colors using this mock?
<details>
{
  "data": [
   {
      "type": "scatter3d",
      "x": [0, 0, 0],
      "y": [1, 1, 1],
      "z": [1, 2, 3],
      "mode": "lines+markers",
      "line": {
        "width": 10,
        "color": [0, 1, 2]
      }
    },
    {
      "type": "scatter3d",
      "x": [0, 0, 0],
      "y": [2, 2, 2],
      "z": [1, 2, 3],
      "mode": "lines+markers",
      "line": {
        "width": 10,
        "color": [0, 1, 2]
      },
      "marker": {
        "color": [0, 1, 2]
      }
    },
    {
      "type": "scatter3d",
      "x": [0, 0, 0],
      "y": [3, 3, 3],
      "z": [1, 2, 3],
      "mode": "lines+markers",
      "marker": {
        "color": [0, 1, 2]
      }
    },
    {
      "type": "scatter3d",
      "x": [1, 1, 1],
      "y": [0, 0, 0],
      "z": [1, 2, 3],
      "mode": "lines+markers",
      "line": {
        "width": 10,
        "colorscale": "Blues",
        "color": [0, 1, 2]
      },
      "marker": {
        "colorscale": "Blues",
        "color": [0, 1, 2]
      }
    },
    {
      "type": "scatter3d",
      "x": [2, 2, 2],
      "y": [0, 0, 0],
      "z": [1, 2, 3],
      "mode": "lines+markers",
      "line": {
        "width": 10,
        "colorscale": "RdBu",
        "color": [0, 1, 2]
      },
      "marker": {
        "colorscale": "RdBu",
        "color": [0, 1, 2]
      }
    },
    {
      "type": "scatter3d",
      "x": [3, 3, 3],
      "y": [0, 0, 0],
      "z": [1, 2, 3],
      "mode": "lines+markers",
      "line": {
        "width": 10,
        "colorscale": "BuRd",
        "color": [0, 1, 2]
      },
      "marker": {
        "colorscale": "BuRd",
        "color": [0, 1, 2]
      }
    }

  ]
}
</details>

Reply:
Ha good catch! That one is already tracked in -> https://github.com/plotly/plotly.js/issues/2953

Leaving that for a future PR.

Reply:
Improved mock in https://github.com/plotly/plotly.js/pull/3341/commits/596bb47bef85d1a4c5e20653602786164f9f89d9,

legend item style improvements are currently tracked in: https://github.com/plotly/plotly.js/issues/581